website/dynamo.py

```diff
@@ -689,6 +764,33 @@ def _decode(self, data):
         return {k: replace_decimals(DDB_DESERIALIZER.deserialize(v)) for k, v in data.items()}
 
 
+class PaginationKey:
```

Comment:
Could you share a doc where this is explained? I don't entirely get why we extract keys, and how is the `extract_ordered` method _ordering_ them? In what direction?

Reply:
Good idea!